MozillaReality/FirefoxReality/app/src/main/res/values/strings.xml

```diff
@@ -777,7 +846,29 @@
     <!-- This string is displayed in a button that when pressed allows a user to view a page that has been blocked from being displayed.-->
     <string name="performance_unblock_page">Unblock Page</string>
     <!-- This string is displayed as the title of a dialog displayed when poor web page performance has been detected. -->
-    <string name="performance_title">Poor Performance Detected</string>
+    <string name="performance_title">Poor Site Performance</string>
     <!-- This string is displayed as the message body of a dialog displayed when poor web page performance has been detected and the page has been unloaded to restore performance. -->
-    <string name="performance_message">The current web page is affecting performance and has been suspended. Reducing the window size can help improve a web page with poor performance.</string>
+    <string name="performance_message">Try reducing the window size to improve performance. Unblocking this page may crash your app.</string>
+
+    <!-- This string is displayed anywhere where we need to display the "Default" string by itself. -->
+    <string name="tag_default">Default</string>
+
+    <!-- This string is displayed in the title of the Voice Search samples collect dialog. That dialog is shown the first time
+         a user tries to use the voice search. -->
+    <string name="voice_samples_collect_dialog_title">Allow Firefox to Anonymously Collect Voice Samples?</string>
+
+    <!-- This string is displayed in the description of the Voice Search samples collect dialog. That dialog is shown the first time
+         a user tries to use the voice search. -->
+    <string name="voice_samples_collect_dialog_description">Your data is always anonymous and protected under our Personal Data Promise. We collect voice samples to improve the experience and performance of our products. You’ll always be able to use voice search, even if you don’t allow collection.&lt;a href=&quot;privacy&quot;&gt;Learn More&lt;/a&gt;</string>
+
+    <!-- This string is displayed in the button of the Voice Search samples collect dialog.
+         When clicked the user opts out from the voice data collection. -->
```

Comment:
Shouldn't this be "opts in" since the string is "Allow"?

Reply:
Fixed